README.md

```diff
@@ -1,18 +1,22 @@
 # Faraday
 
 Faraday is an HTTP client lib that provides a common interface over many
-adapters (such as Net::HTTP) and embraces the concept of Rack middleware when
+adapters (such as `Net::HTTP`, the default adapter) and embraces the concept of Rack middleware when
 processing the request/response cycle.
 
 Faraday supports these adapters:
 
-* Net::HTTP
+* [`Net::HTTP`][net_http]
+* [`Net::HTTP::Persistent`][net_http_persistent] _(experimental)_
```

Comment:
The "experimental" note here isn't necessary. Net::HTTP::Persistent works pretty well in my experience. Users should switch between adapters at their own risk.